src/components/QueryOutput/QueryOutput.js

```diff
@@ -1876,6 +1881,17 @@ export default class QueryOutput extends React.Component {
     const { displayType } = this.state
     const { queryResponse } = this.props
 
+    if (
+      getAuthentication(this.props.authentication).isQandA &&
```

Comment:
Should use _.get(getAuthentication(this.props.authentication), isQandA) to be safe

Reply:
done